react-intro/react-intro-quiz-notes.md

```diff
@@ -0,0 +1,45 @@
+# react-intro-quiz-notes
+
+## Quiz Questions
+
+After completing this exercise, you should be able to discuss or answer the following questions:
+
+- What is React?
+  A javascript library/framework for building web apps.
+
+- What is a React component?
+  A component is a piece of the UI (user interface) that has its own logic and appearance.
+
+- How do you mount a React app (root component) to the DOM?
+  By adding the keywords `export default` to the beginning of the main component definition line.
```

Comment:
Not quite! Take a look at `index.js`. How are we actually injecting our React application into our HTML file?